src/sentry/static/sentry/app/components/events/meta/annotatedText.jsx

```diff
@@ -36,15 +36,31 @@ const ErrorIcon = styled(InlineSvg)`
 const REMARKS = {
   a: 'Annotated',
   x: 'Removed',
-  s: 'Substituted',
+  s: 'Replaced',
   m: 'Masked',
   p: 'Pseudonymized',
   e: 'Encrypted',
 };
 
+const KNOWN_RULES = {
+  '!limit': 'size limits',
+  '!raw': 'raw payload',
+  '!config': 'SDK configuration',
+};
+
+function getTooltipText(remark, rule) {
+  remark = REMARKS[remark];
```

Comment:
nit: we should avoid reassigning an argument (it's harmless here, but good practice) -- we should have an eslint rule for this too

Reply:
Fixed